### PR TITLE
Seed fo real

### LINKED ISSUE #33: seed with users from NS
from staging 

store that info in cache

### PR DESCRIPTION
**Pull request by angaither**
#### What's this PR do?
seeds admins and users from stage db. 

#### How should this be manually tested?
run `php artisan db:seed --class=UserTableSeeder` to see users added

#### Any background context you want to provide?
this is to help build #27
need have Northstar-QA values set up in `.env` file for this to work

#### What are the relevant tickets?
Fixes #33 

cc @sbsmith86

### CODE REVIEW

File: database/seeds/UserTableSeeder.php [hunk 1]
```diff
@@ -3,10 +3,30 @@
 use Gladiator\Models\User;
 use Gladiator\Models\WaitingRoom;
 use Illuminate\Database\Seeder;
+use Gladiator\Console\Commands;
```
Comment (weerd) [inline database/seeds/UserTableSeeder.php]: I don't think you need this...

Comment (deadlybutter) [inline database/seeds/UserTableSeeder.php]: StyleCI confirms as well

File: database/seeds/UserTableSeeder.php [hunk 2]
```diff
@@ -14,21 +34,30 @@ class UserTableSeeder extends Seeder
...
-            User::create([
-                'id' => str_random(24),
-                'role' => 'admin',
-            ]);
+        $admins = [
+            'agaither@dosomething.org',
+            'ssmith@dosomething.org',
+            'mfantini@dosomething.org',
+            'dlorenzo@dosomething.org',
+            'jkent@dosomething.org',
```
Comment (deadlybutter) [inline database/seeds/UserTableSeeder.php]: Ive been using joe@ds not jkent

Comment (weerd) [inline database/seeds/UserTableSeeder.php]: The one `joe` to rule them all.

**Comment by weerd** [comment]
Still some StyleCI fixes needed, but otherwise :+1:

**Comment by weerd** [comment]
:+1: :dancer:

**Pull request merged**